MaterialLib/Fluid/PropertyVariableType.h

```diff
@@ -0,0 +1,34 @@
+/**
+ *  \copyright
+ *   Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *              Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * \file   PropertyVariableType.h
+ *
+ * Created on November 29, 2016, 2:28 PM
+ */
+
+#ifndef OGS_PROPERTY_VARIABLE_TYPE_H
+#define OGS_PROPERTY_VARIABLE_TYPE_H
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+/// Variable that determine the property.
+enum class PropertyVariableType
+{
+    T = 0,                   ///< temperature.
+    p = 1,                   ///< pressure.
+    rho = p,                 ///< density. For some models, rho substitutes p
```

Comment:
It seems to me this is excluding material models where both rho and p are used, isn't it?
Anyway relying on actual enumeration numbers is not a good idea. To be discussed.